crates/subspace-node/src/import_blocks_from_dsn.rs

```diff
@@ -48,27 +52,30 @@ pub struct ImportBlocksFromDsnCmd {
 
 impl ImportBlocksFromDsnCmd {
     /// Run the import-blocks command
-    pub async fn run<B, C, IQ>(
+    pub async fn run<IQ>(
         &self,
-        client: Arc<C>,
+        client: Arc<FullClient<RuntimeApi, ExecutorDispatch>>,
```

Comment:
Add another trait to `where` clause below to be able to call `.chain_info()` instead of hardcoding client type.